src/test/java/net/pms/configuration/FormatConfigurationRegressionTest0.java

```diff
@@ -10,966 +10,699 @@
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "MIMETYPE_AUTO"+ "'", str0.equals("MIMETYPE_AUTO"));
-
   }
 
   @Test
   public void test42() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test42"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.VP7;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "vp7"+ "'", str0.equals("vp7"));
-
   }
 
   @Test
   public void test43() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test43"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.THREEGPP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "3gp"+ "'", str0.equals("3gp"));
-
   }
 
   @Test
   public void test44() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test44"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MJPEG;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mjpeg"+ "'", str0.equals("mjpeg"));
-
   }
 
   @Test
   public void test45() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test45"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.BMP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "bmp"+ "'", str0.equals("bmp"));
-
   }
 
   @Test
   public void test46() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test46"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.TTA;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "tta"+ "'", str0.equals("tta"));
-
   }
 
   @Test
   public void test47() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_LC;
 
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test47"); }
+    // Regression assertion (captures the current behavior of the code)
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-lc"+ "'", str0.equals("aac-lc"));
+  }
 
-    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC;
+  @Test
+  public void testAACMain() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_MAIN;
 
     // Regression assertion (captures the current behavior of the code)
-    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac"+ "'", str0.equals("aac"));
-
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-main"+ "'", str0.equals("aac-main"));
```

Comment:
Honestly, i never say it, but i never understood that point. What use for ?
For me it's just look as a dead appendice, as MediaInfo don't recognize it ? So that will be never used. Though i can be wrong.